docs/logging/harvester-logging.md

```diff
@@ -36,9 +36,9 @@ For Harvester clusters upgraded from version v1.1.x, the logging feature is conv
 
 ## High-level Architecture
 
-The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers both `Harvester` and `Rancher` as an in-house logging solution.
+The [Logging Operator](https://kube-logging.dev/docs/#overview) now powers both `Harvester` and `Rancher` as an in-house logging solution.
```

Comment:
We need to update those links in v1.2 doc, right ?

Reply:
v1.2 may be EOL. Please @jillian-maroket confirm, should we update v1.2 part as well? thanks.

Reply:
According to the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-2-x/), the EOL date for v1.2.x is 08 September 2024.

@bk201 Please confirm so we can finalize the scope of this PR. I'll also add the `EOL` label to v1.2.0 in the header's version selector.

Reply:
Let's just update the non-EOL doc, starting from v1.3